docs/production-deployment/self-hosted-guide/checklist.mdx

```diff
@@ -46,106 +77,81 @@ At a high level, you will want to track these 3 categories of metrics:
-We support protobuf with built-in converters.
-
-### Dealing with very long-lived or very event-heavy Workflows
-
-If you know that your Workflow will live for a very long time or generates a large volume of [Events](/workflows#event), you need to deal with the Event [limits](/workflows#limits) and [Continue-As-New](/workflows#continue-as-new) at some point.
-
-One technique to prepare for this eventuality (and at the same time make your life easier when you need to make changes to the [Workflow Definition](/workflows#workflow-definition)) is to structure your Workflow as an "entity Workflow."
-
-### Make all non-transient state serializable
-
-You already defined your Workflow input as an object or struct like we mentioned in the previous section, right?
-If you didn't, do that.
-
-The struct should be capable of representing all state that your Workflow cares about.
-Because it is also your Workflow input, it must be serializable.
+## Availability
 
-In practice, this often ends up looking like a state machine.
+A high level of availability and reliability (99.99%) is a requirement for mission critical deployments.
+We recommend testing for this availability level while load testing.
+We also recommend validating this level of reliability while doing server upgrades, to ensure no loss of service availability.  
 
-### Continue-As-New on demand
+Temporal Clusters can be deployed in as many regions as needed to meet various requirements:
+- Data Residency
+- Latency
+- Security / Isolation
+This can multiply the effort to implement and maintain clusters.
 
-Set up a mechanism to force your Workflow to Continue-As-New on demand.
-The easiest way to do this is to define a Signal handler that continues the Workflow when received.
-Be sure that all previously received Signals have been processed and that all running [Child Workflows](/encyclopedia/child-workflows) have been waited on or otherwise dealt with.
+[Temporal Cloud is available in 15 AWS regions](/cloud/service-availability).
 
-The newly started Workflow then unpacks the state in the input and can "resume" the Workflow according to your business logic.
+## Management and Control Plane 
 
-A huge benefit of this approach is that you avoid making patches to running Workflows.
-Instead, tell them all to continue onto a new [Task Queue](/workers#task-queue) with [Workers](/workers) that have the updated Workflow code.
+Temporal success leads to larger Temporal deployments.
+Needs can increase, and can go from having one or two production use cases in a single region to many use cases in many regions.
+Running multiple clusters is complex work, as each needs its own setup, tuning, and configuration. 
 
-### When and how to make changes to your Workflow code
+This operational management pain leads to a need to monitor and manage all of your Temporal Services in a unified way.
```

Comment:
Maybe flip the last half and the first half of the sentence? Wording suggestions: 

> "Needing to monitor and manage all your Temporal Services in a unified way leads to operational management pain. Consider adding a layer on top of Temporal to manage multiple Temporal Services: a control plane. A control plane manages and directs data flow, deciding where data packets should be sent. A Temporal Service data plan can streamline operations and improve efficiency. Since Temporal does not ship its own open source data plane, rolling your own can be complex and take effort to add." 

But then there's no suggestions about how to find or do this or even where to look for solutions.